src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double base;
+    private double height;
+
+    public RightTriangle(String color, double base, double height) {
+        super(color);
+        this.base = base;
+        this.height = height;
+    }
+
+    public RightTriangle(String color) {
+        super(color);
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * base * height;
+    }
+
+    @Override
+    public void draw() {
```

Comment:
The `draw()` method should be separated into its own interface, as the `draw()` and `getArea()` methods are conceptually different from each other. Consider creating and implementing a separate `Drawable` interface for the `draw()` method.